model/metrics/clr-metrics.yaml

```diff
@@ -0,0 +1,166 @@
+    attributes:
+      - ref: clr.gc.generation
+        requirement_level: required
+
+  - id: metric.clr.gc.objects.size
+    type: metric
+    metric_name: clr.gc.objects.size
+    brief: "The number of bytes currently allocated on the managed GC heap. Fragmentation and other GC committed memory pools are excluded."
+    instrument: updowncounter
+    unit: "By"
+    stability: experimental
+
+  - id: metric.clr.gc.allocations.size
+    type: metric
+    metric_name: clr.gc.allocations.size
+    brief: >
+      The number of bytes allocated on the managed GC heap since the process
+      started. The returned value does not include any native allocations.
+    instrument: counter
+    unit: "By"
+    stability: experimental
+
+  - id: metric.clr.gc.committed_memory.size
+    type: metric
+    metric_name: clr.gc.committed_memory.size
+    brief: >
+      The amount of committed virtual memory for the managed GC heap, as
+      observed during the latest garbage collection.
+    note: >
+      Committed virtual memory may be larger than the heap size because it
+      includes both memory for storing existing objects (the heap size) and some
+      extra memory that is ready to handle newly allocated objects in the
+      future.
+    instrument: updowncounter
+    unit: "By"
+    stability: experimental
+
+  - id: metric.clr.gc.heap.size
+    type: metric
+    metric_name: clr.gc.heap.size
+    brief: >
+      The heap size (including fragmentation), as observed during the latest
+      garbage collection.
+    instrument: updowncounter
+    unit: "By"
+    stability: experimental
+    attributes:
+      - ref: clr.gc.generation
+        requirement_level: required
+
+  - id: metric.clr.gc.heap.fragmentation.size
+    type: metric
+    metric_name: clr.gc.heap.fragmentation.size
+    brief: >
+      The heap fragmentation, as observed during the latest garbage collection.
+    instrument: updowncounter
+    unit: "By"
+    stability: experimental
+    attributes:
+      - ref: clr.gc.generation
+        requirement_level: required
+
+  - id: metric.clr.gc.duration
+    type: metric
+    metric_name: clr.gc.duration
+    brief: "The total amount of time paused in GC since the process started."
+    instrument: counter
+    unit: "ns"
+    stability: experimental
+  - id: metric.clr.jit.il_compiled.size
+    type: metric
+    metric_name: clr.jit.il_compiled.size
+    brief: "Count of bytes of intermediate language that have been compiled since the process start."
+    instrument: counter
+    unit: "By"
+    stability: experimental
+
+  - id: metric.clr.jit.methods_compiled.count
+    type: metric
+    metric_name: clr.jit.methods_compiled.count
+    brief: >
+      The number of times the JIT compiler (re)compiled methods since the
+      process start.
+    instrument: counter
+    unit: "{method}"
+    stability: experimental
+
+  - id: metric.clr.jit.compilation_time
+    type: metric
+    metric_name: clr.jit.compilation_time
```

Comment:
Shouldn't we follow the `[entity.time]` [naming guideline](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metrics.md#instrument-naming) here as well? 

So maybe `clr.jit_compilation.time` or `clr.jit.compilation.time`?

Reply:
Yes, makes sense. I'll update this one.